qemu/tests/cfg/vsock.cfg

```diff
@@ -2,6 +2,7 @@
     no Windows
     type = vsock_test
     vsocks = vhost_vsock0
+    virtio_dev_iommu_platform = on
```

Comment:
Remove this line since it‘s personal usage.

Reply:
solved